### PR TITLE
Replace Memgraph Platform image with Memgraph MAGE image where applicable

### PR DESCRIPTION
**Pull request by kgolubic**
### Description

Replaced Memgraph Platform image with Memgraph MAGE image where it was applicable.

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page

### Related PRs and issues

PR this doc page is related to: 
(especially necessary if the PR is related to a release)

Closes:
https://github.com/memgraph/documentation/issues/706

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/configuration/replication.mdx [hunk 1]
```diff
@@ -313,19 +313,19 @@ you need expose different ports for each instance.
...
 ```
-docker run -p 6788:6787 -p 3001:3000 memgraph/memgraph-platform
+docker run -p 6788:6787 memgraph/memgraph-mage
 ```
 
 REPLICA instance 2:
 
 ```
-docker run -p 6789:6787 -p 3002:3000 memgraph/memgraph-platform
+docker run -p 6789:6787 memgraph/memgraph-mage
```
Comment (katarinasupe) [inline pages/configuration/replication.mdx:328]: Isn't it 7687 port internally?

File: pages/configuration/ssl-encryption.mdx [hunk 2]
```diff
@@ -190,11 +181,8 @@ authenticated and encrypted connection to a database instance.
 5. [Stop the Docker container](/getting-started/install-memgraph/docker#stop-container),
    then start it again, including the volumes you used in step 1.
 
-   If you are running `memgraph-platform` image, pass the configuration flag
-   <code>MGCONSOLE="--use-ssl=true"</code>:
-
    ```
-   docker run -p 7687:7687 -p 3000:3000 -p 7444:7444 -v mg_lib:/var/lib/memgraph -v mg_etc:/etc/memgraph -e MGCONSOLE="--use-ssl=true" memgraph/memgraph-platform
+   docker run -p 7687:7687 -p 7444:7444 -p 3000:3000 -v mg_lib:/var/lib/memgraph -v mg_etc:/etc/memgraph memgraph/memgraph-mage --use-ssl=true
```
Comment (katarinasupe) [inline pages/configuration/ssl-encryption.mdx:185]: Is port 3000 necessary here?

File: pages/getting-started/first-steps-with-docker.mdx [hunk 3]
```diff
@@ -264,41 +264,10 @@ your database tasks. From a single image, you can spawn multiple containers.
...
-[MAGE](/advanced-algorithms) library.
-
-To download the latest Memgraph Platform image, run:
-
-```
-docker pull memgraph/memgraph-platform
-```
-
-All Memgraph Docker images are availble at [Memgraph's Docker
+the Docker image. All Memgraph Docker images are availble at [Memgraph's Docker
```
Comment (katarinasupe) [inline pages/getting-started/first-steps-with-docker.mdx:267]:
```suggestion
the Docker image. All Memgraph Docker images are available at [Memgraph's Docker
```

File: pages/getting-started/first-steps-with-docker.mdx [hunk 4]
```diff
@@ -329,24 +297,19 @@ The most common ports published while running Memgraph are:
...
   version 2.0 and newer)
 
-For example, if you are running two instances using the `memgraph-platform`
-image and you want to connect to both instances using the Memgraph Lab
-in-browser application, you would run the first instance with:
+For example, if you are running two instances using the `memgraph-mage`
+image and you would run the first instance with:
 
-<CodeSnippet code="docker run -p 7444:7444 -p 3000:3000 memgraph/memgraph-platform" page="getting-started/first-steps-with-docker" os="docker"/>
+<CodeSnippet code="docker run -p 7444:7444 -p 6787:6787 memgraph/memgraph-mage" page="getting-started/first-steps-with-docker" os="docker"/>
```
Comment (katarinasupe) [inline pages/getting-started/first-steps-with-docker.mdx:306]: wrong port

File: pages/getting-started/first-steps-with-docker.mdx [hunk 5]
```diff
@@ -329,24 +297,19 @@ The most common ports published while running Memgraph are:
...
 
-Because port `7444` and `3000` are now taken, you need to change the left side
+Because ports `7444` and `6787`are now taken, you need to change the left side
```
Comment (katarinasupe) [inline pages/getting-started/first-steps-with-docker.mdx:308]: wrong port

File: pages/getting-started/first-steps-with-docker.mdx [hunk 6]
```diff
@@ -329,24 +297,19 @@ The most common ports published while running Memgraph are:
...
 ports (host ports):
 
-<CodeSnippet code="docker run -p 7445:7444 -p 3001:3000 memgraph/memgraph-platform" page="getting-started/first-steps-with-docker" os="docker"/>
+<CodeSnippet code="docker run -p 7445:7444 -p 6788:6787 memgraph/memgraph-mage" page="getting-started/first-steps-with-docker" os="docker"/>
```
Comment (katarinasupe) [inline pages/getting-started/first-steps-with-docker.mdx:311]: wrong port

File: pages/getting-started/first-steps-with-docker.mdx [hunk 7]
```diff
@@ -511,7 +464,7 @@ You should get an output similar to this:
 
 ```console
 CONTAINER ID   IMAGE                        COMMAND                  CREATED        STATUS        PORTS                                                                    NAMES
-45fa0f86f826   memgraph/memgraph-platform   "/bin/sh -c '/usr/bi…"   21 hours ago   Up 21 hours   0.0.0.0:3000->3000/tcp, 0.0.0.0:7444->7444/tcp, 0.0.0.0:7687->7687/tcp   admiring_almeida
+45fa0f86f826   memgraph/memgraph-mage       "/bin/sh -c '/usr/bi…"   21 hours ago   Up 21 hours   0.0.0.0:3000->3000/tcp, 0.0.0.0:7444->7444/tcp, 0.0.0.0:7687->7687/tcp   admiring_almeida
```
Comment (katarinasupe) [inline pages/getting-started/first-steps-with-docker.mdx:467]: Run again and put the correct info from console (this includes unused ports)

**Review by katarinasupe — CHANGES_REQUESTED** [review]

File: pages/configuration/replication.mdx (hunk 1 above)
Comment (kgolubic) [reply pages/configuration/replication.mdx:328]: You are right.

**Pull request merged**